### PR TITLE
TP-11901; TP-11733; TP-11734 - Update JSON, Kaminari and Rack Gems

### PR DESCRIPTION
**Pull request by thenapking**
[TP-11901](https://maps.tpondemand.com/entity/11901-update-rack-to-version-greater220)
[TP-11733](https://maps.tpondemand.com/entity/11733-resolve-kaminari-gem-security-warnings)
[TP-11734](https://maps.tpondemand.com/entity/11734-upgrade-json-gem-in-frontend-project)

Update to gem file, inc websocket-extensions
Fix CMS class as now test failed
This fix will leave warnings for CVE-2020-8184 and CVE-2020-8161, which can now be ignored. These relate to Rack, which is patched.

### CODE REVIEW

**Comment by jordanlg27** [comment]
https://maps.tpondemand.com/entity/11901

**Comment by jordanlg27** [comment]
https://maps.tpondemand.com/entity/11734

**Comment by jordanlg27** [comment]
https://maps.tpondemand.com/entity/11733

**Comment by thenapking** [comment]
Before approving this PR we should also consider whether we would add the script referenced here: [ActionView Vulnerability](https://github.com/moneyadviceservice/frontend/security/dependabot/Gemfile.lock/actionview/open)
This has been spotted by Dependabot. We cannot move to later versions of ActionView as they are tied to the Rails version.  Our option therefore is to add the script, or if we do not have any vulnerable code in the JS, to ignore this warning.

File: Gemfile [hunk 1]
```diff
@@ -55,28 +55,30 @@ gem 'nunes'
...
 gem 'adal', git: 'git@github.com:moneyadviceservice/azure-activedirectory-library-for-ruby'
 gem 'cream', '2.1.8'
 gem 'dough-ruby', git: 'git@github.com:moneyadviceservice/dough.git', branch: '11691_DAL_Current-location-warning-message_v5.41', ref: 'a90d79d'
-gem 'mas-cms-client', '1.20.0'
+gem 'mas-cms-client', '1.20.1', git: 'git@github.com:moneyadviceservice/mas-cms-client.git', branch: 'TP-11734-upgrade-json'
 gem 'site_search', '0.3.0'
 # Tools
 gem 'action_plans', '~> 5.5.0'
-gem 'advice_plans', '~> 4.1.0'
+gem 'advice_plans', '~> 4.1.1', git: 'git@github.com:moneyadviceservice/advice_plans.git', branch: 'TP-11733-update-kaminari'
```
Comment (davidtrussler) [inline Gemfile:81]: Will the references to the `advice_plans` and `mas-cms-client` gems be updated to the new versions once they are merged?

Comment (thenapking) [reply Gemfile:81]: Yes, they will once this has had QA approval and I've merged the tools' feature branches.  I didn't want to do that prematurely.

**Pull request merged**